worlds/bomb_rush_cyberfunk/docs/setup_en.md

```diff
@@ -0,0 +1,41 @@
+# Bomb Rush Cyberfunk Multiworld Setup Guide
+
+## Quick Links
+
+- Bomb Rush Cyberfunk: [Steam](https://store.steampowered.com/app/1353230/Bomb_Rush_Cyberfunk/)
+- Archipelago Mod: [Thunderstore](https://thunderstore.io/c/bomb-rush-cyberfunk/p/TRPG/Archipelago/), 
+[GitHub](https://github.com/TRPG0/BRC-Archipelago/releases)
+
+## Setup
+
+To install the Archipelago mod, you can use a mod manager like 
+[r2modman](https://thunderstore.io/c/bomb-rush-cyberfunk/p/ebkr/r2modman/), or install manually by following these steps:
+
+1. Download and install [BepInEx 5.4.22 x64](https://github.com/BepInEx/BepInEx/releases/tag/v5.4.22) in your Bomb Rush 
+Cyberfunk root folder. *Do not use any pre-release versions of BepInEx 6.*
+
+2. Start Bomb Rush Cyberfunk once so that BepInEx can create its required configuration files.
+
+3. Download the zip archive from the [releases](https://github.com/TRPG0/BRC-Archipelago/releases) page, and extract its 
+contents into `BepInEx\plugins`.
+
+After installing Archipelago, there are some additional mods that can also be installed for a better experience:
```

Comment:
[Optional] Might be best to separate this as another heading so it doesn't feel necessary for setup. Stardew valley lists mods as "Optional Software" as an example.

Reply:
I'm choosing to leave this as is